src/payform.coffee

```diff
@@ -167,6 +167,10 @@
   cardFromType = (type) ->
     return card for card in payform.cards when card.type is type
 
+  getDirectionality = (target) ->
+    style = getComputedStyle(target)
+    return if style then style['direction'] else document.dir
```

Comment:
The fallback is only necessary on Firefox for some edge cases (such as a `display: none` iframe), https://bugzilla.mozilla.org/show_bug.cgi?id=1467722.

However, starting from Firefox 62, in the aforementioned edge cases Firefox no longer returns `null`: https://www.fxsitecompat.com/en-CA/docs/2018/getcomputedstyle-no-longer-returns-null-when-style-can-t-be-retrieved/

Instead, it returns an object with `length: 0` and an empty string for every declaration.

For Firefox 62+ compatibility, this should be something like:

```coffeescript
# Work around Firefox not returning the styles in some edge cases.
style and style['direction'] or document.dir
```